packages/web-components/fast-foundation/src/tabs/tabs.ts

```diff
@@ -159,22 +176,24 @@ export class Tabs extends FASTElement {
                     "aria-controls",
                     typeof tabpanelId !== "string" ? `panel-${index + 1}` : tabpanelId
                 );
-                tab.setAttribute(
-                    "style",
-                    this.isHorizontal()
-                        ? `grid-column: ${index + 1};`
-                        : `grid-row: ${index + 1};`
-                );
                 tab.addEventListener("click", this.handleTabClick);
                 tab.addEventListener("keydown", this.handleTabKeyDown);
                 tab.setAttribute("tabindex", this.activeTabIndex === index ? "0" : "-1");
                 if (this.activeTabIndex === index) {
                     this.activetab = tab;
                 }
-                !this.isHorizontal()
-                    ? tab.classList.add("vertical")
-                    : tab.classList.remove("vertical");
+            } else {
+                this.showActiveIndicator = false;
             }
+            tab.setAttribute(
```

Comment:
As far as I can tell this is an existing pattern if I'm not mistaken. Does this need to be addressed in this PR or can we make a new ticket to handle this change?

Reply:
We can do it with a new ticket. Just want to be careful not to do things that will interfere with user styles. So, in this case, it's better to set the individual style properties rather than overwrite the entire style object.

Reply:
@litteredwitherrors We should not override the entire style attribute, if you are seeing this elsewhere we should fix.
There is a difference between.

```
tab.setAttribute("style", `grid-column: ${index + 1}`)
```

and
```
tab.style.gridColumn = `${index + 1}`
```

The latter will append the grid column to the style attribute vs `setAttribute` which will wipe the entire style object.

You should follow this pattern:
```
const gridProperty: string = this.isHorizontal() ? "gridColumn" : "gridRow";

tab.style[gridProperty] = `${index + 1}`;
```

Reply:
Applied suggestions from @eljefe223.